rlbot/interface.py

```diff
@@ -250,43 +250,46 @@ def run(self, *, background_thread: bool = False):
         else:
             self._running = True
             while self._running and self.is_connected:
-                self._running = self.handle_incoming_messages(blocking=True)
+                self._running, _ = self.handle_incoming_messages(blocking=True)
             self._running = False
 
-    def handle_incoming_messages(self, blocking: bool = False) -> bool:
+    def handle_incoming_messages(self, blocking: bool = False) -> tuple[bool, bool]:
```

Comment:
I highly recommend using an enum instead. There are only three cases and they also become more readable. E.g. like I did it https://github.com/VirxEC/csharp-interface/pull/7